.ci/matrix_job.yaml

```diff
@@ -163,7 +164,7 @@ steps:
     enable: ${do_build}
     run: |
       [ "x${do_build}" == "xtrue" ] && action=yes || action=no
-      echo "COMPILE_DOCA=$COMPILE_DOCA COMPILE_DPCP=$COMPILE_DPCP"
+      echo "COMPILE_DOCA=$COMPILE_DOCA \n COMPILE_DPCP=$COMPILE_DPCP"
```

Comment:
what is this? why you need to echo? i also dont expect to see this change here in gtest disable commit